src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -908,6 +908,12 @@
         - name: Create a load balancer on a specific virtual network and subnet.
           text: >
             az network lb create -g MyResourceGroup -n MyLb --vnet-name MyVnet --subnet MySubnet
+        - name: create a zone flavored public facing load balancer through provisiong a zonal public ip
```

Comment:
Typo: `provisiong`

Reply:
Zone flavored?  What does that taste like? 🍨

Reply:
We don't want to say `zonal lb` which is conceptual wrong, but LB does have some relationships with zone. The `flavor` is the word I picked when I chatted with NRP folks.